ftd/t/js/80-or-type-constant.html

```diff
@@ -0,0 +1,126 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <meta charset="UTF-8">
+    
+    <meta content="fastn" name="generator">
+    
+    
+    <script>
+        let __fastn_package_name__ = "foo";
+    </script>
+
+    <script src="fastn-js.js"></script>
+    
+
+    <style>
+       
+    </style>
+</head>
+<meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1, user-scalable=0">
+<body data-id="1"><div data-id="2" class="ft_column __w-1 __h-2"><div data-id="3">Smol</div><div data-id="4">Smol</div></div></body><style id="styles">
+    .__w-1 { width: 100%; }
+	.__h-2 { height: 100%; }
+    </style>
+<script>
+    (function() {
+        let main = function (parent) {
+  let __fastn_super_package_name__ = __fastn_package_name__;
+  __fastn_package_name__ = "foo";
+  try {
+    let parenti0 = fastn_dom.createKernel(parent, fastn_dom.ElementKind.Text);
+    parenti0.setProperty(fastn_dom.PropertyKind.StringValue, fastn.formula([global.foo__b,
+    global.foo__small], function () {
+      if (function () {
+        return (fastn_utils.getter(global.foo__b) == fastn_utils.getter(global.foo__small));
+      }()) {
+        return "Smol";
+      } else {
+        return "!Smol";
+      }
+    }
+    ), inherited);
+    let parenti1 = foo__fancy_button(parent, inherited, {
+      bt: global.foo__b
+    });
+  } finally {
+    __fastn_package_name__ = __fastn_super_package_name__;
+  }
+}
+global["main"] = main;
+fastn_utils.createNestedObject(global, "foo__button_type", fastn.recordInstance({
+  small: 1,
+  medium: 2,
+  large: 3
+}));
+fastn_utils.createNestedObject(global, "foo__b", 1);
+fastn_utils.createNestedObject(global, "foo__small", 1);
```

Comment:
```suggestion
fastn_utils.createNestedObject(global, "foo__b", global.foo__button_type.get("small"));
```

Can this be done instead?

Also instead of creating a new variable `global.foo__small`, can we do `global.foo__button_type.get("small")`?